web/beacon-app/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,138 @@
+              <ul>
+                <li>
+                  <a href="#">Go</a>
+                </li>
+                <li>
+                  <a href="#">Python</a>
+                </li>
+              </ul>
+            </div> */}
+          </div>
+          <div className="border-t py-6">
+            <div className="ml-20 grid grid-cols-4">
+              <div className="hover:max-w-[50%] hover:rounded-full hover:bg-icon-hover">
+                <a
+                  href="https://twitter.com/rotationalio"
+                  className="flex items-center"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img src={twitter} alt="Twitter logo" className="" />
+                  <span className="ml-4">Twitter</span>
+                </a>
+              </div>
+              <div className="hover:max-w-[50%] hover:rounded-full hover:bg-icon-hover">
+                <a
+                  href="https://github.com/rotationalio"
+                  className="flex items-center"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img src={github} alt="GitHub logo" className="" />
+                  <span className="ml-4">GitHub</span>
+                </a>
+              </div>
+              <div className="hover:max-w-[50%] hover:rounded-full hover:bg-icon-hover">
+                <a
+                  href="https://www.linkedin.com/company/rotational"
+                  className="flex items-center"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img src={linkedin} alt="LinkedIn logo" className="" />
+                  <span className="mt-2 ml-4">LinkedIn</span>
+                </a>
+              </div>
+              <div className="mt-2 hover:max-w-[50%] hover:rounded-full hover:bg-icon-hover">
+                <a
+                  href="mailto:info@rotational.io"
+                  className="flex items-center"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img src={email} alt="Envelope" className="" />
+                  <span className="ml-4">Email</span>
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="mt-4 justify-between py-10 text-white sm:flex">
+            <p className="">
+              Copyright © {new Date().getFullYear()} Rotational Labs, Inc · All Rights Reserved
+            </p>
+            <ul className="mt-4 flex sm:mt-0">
+              <li className="mr-4 border-r pr-4">
+                <a href="https://rotational.io/privacy">Privacy Policy</a>
+              </li>
+              <li className="">
+                <a href="https://rotational.io/terms">Terms of Use</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </footer>
+  );
+}
+
+export default Footer;
```

Comment:
```suggestion
export default React.memo(Footer);
```
Would you mind making this component a pure component?

Reply:
I got an error with this change so, I imported memo and did the following `export default memo(LandingFooter);`. I also updated the file name to be a little more specific since this footer won't be used once a user registers or logs in.